module/dice/basic-roll.mjs

```diff
@@ -75,6 +77,8 @@ export default class BasicRoll extends Roll {
    */
   static fromConfig(config, process) {
     const formula = (config.parts ?? []).join(" + ");
+    config.options ??= {};
+    config.options.target ??= process.target;
```

Comment:
Are we passing `target` as a `process` parameter somewhere? Should we not be? It seems it's not documented in `BasicRollProcessConfiguration`.

Reply:
Should be part of the typedef for process:

```javascript
/**
 * Configuration data for the process of creating one or more basic rolls.
 *
 * @typedef {object} BasicRollProcessConfiguration
 * @property {BasicRollConfiguration[]} rolls  Configuration data for individual rolls.
 * @property {Event} [event]                   Event that triggered the rolls.
 * @property {string[]} [hookNames]            Name suffixes for configuration hooks called.
 * @property {Document} [subject]              Document that initiated this roll.
 * @property {number} [target]                 Default target value for all rolls.
 */
```